newsService/src/main/java/com/kovan/app/service/NewsService.java

```diff
@@ -80,18 +80,18 @@ public List<NewsDto> getTopHeadlines() {
                 .takeWhile(article -> !parseDate(article.getPublishedAt()).isBefore(yesterday))
                 .collect(Collectors.partitioningBy(article -> parseDate(article.getPublishedAt()).equals(yesterday)));
 
-        List<NewsDto.Article> newsArticles = getListIfNotEmpty(partitionedArticles.get(true));
-        List<NewsDto.Article> futureArticles = getListIfNotEmpty(partitionedArticles.get(false));
+        List<NewsDto.Article> newsArticles = createList(partitionedArticles.get(true));
```

Comment:
Do we really need this createList please think about it.

Reply:
Removed createList method.